packages/executor/src/modules/eth.ts

```diff
@@ -36,9 +36,7 @@ export class Eth {
     private config: NetworkConfig,
     private logger: Logger
   ) {
-    if (
-      ["arbitrum", "arbitrumNitro", "arbitrumNova"].includes(this.networkName)
-    ) {
+    if (["arbitrum", "arbitrumNova"].includes(this.networkName)) {
```

Comment:
I would prefer Skandha to autodetect the availability of the conditional endpoint and use that, rather than us releasing a new version every time a new chain adds support for it.

Reply:
First, this piece of code is used to add additional calculation methods for preverificationGasLimit on L2 chains and is not related to conditional transactions. I removed arbitrumNitro from there because I found out during testing that this chain doesn't need that.

And we don't need to release a new version of Skandha for each network that should use the conditional transactions, it can be turned on and off in the config file or the env variables.


>autodetect the availability of the conditional endpoint

I couldn't find an easy way to detect if the rpc provider supports that endpoint, because each of them has a unique error/success message. I think it's safer to manually check for the support of the conditional transactions and turn it on manually in the config file or via the env variables.


Tested with RPCs from chainlist.org

1.  ============
![image](https://github.com/etherspot/skandha/assets/7760339/d790425f-3a6f-482c-9460-be7bea0a5043)

2. ============
![image](https://github.com/etherspot/skandha/assets/7760339/fd65ef82-ad19-4e70-96e3-20e2bededb6d)

3. ============
![image](https://github.com/etherspot/skandha/assets/7760339/bfd24ced-626f-4ed1-87dc-4a61bbc24294)


3.1 Same RPC as above, but reports a different message: 
![image](https://github.com/etherspot/skandha/assets/7760339/dd64fdca-5b2f-4ac5-9795-3bfd94057909)

4. ============
![image](https://github.com/etherspot/skandha/assets/7760339/a583779c-73c7-48f4-a76b-b1aeb49476e9)

5. ============
![image](https://github.com/etherspot/skandha/assets/7760339/5fd84f03-a026-4399-8d01-d04be10e3a30)

Reply:
Alright. Let us keep an eye out for this and look at it in future.